Makefile

```diff
@@ -39,10 +21,13 @@ example-executor:
 	go build -o ${EXAMPLES}/$@ ${EXAMPLES}/example_executor.go
 
 format:
-	go fmt ${LIBS:%=$(PKG_PREFIX)/%}
+	gofmt -s -w .
+
+formatted:
+	! gofmt -s -d . 2>&1 | read
 
 vet:
-	go vet ${LIBS:%=$(PKG_PREFIX)/%}
+	go vet ./...
 
-test test.v:
-	flags=""; test "$@" != "test.v" || flags="-test.v"; pkg="${TEST}"; test -n "$$pkg" || pkg="${LIBS:%=$(PKG_PREFIX)/%}"; go test $$pkg $$flags
+test: formatted vet
```

Comment:
Why would we provide such functionality through the Makefile when it's implemented by the Go tool? If a Go developer wants to run any of these things independently he'll intuitively pick the Go tool.